src/components/ConnectWallet/Wallets/Joule.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <connection-details name="Joule (Chrome Extension)" requires="lnd">
+    <step-list>
+      <step>
+        Install the <a href="https://chrome.google.com/webstore/detail/joule/aejmoogjdllanidlpfjmmmmimfaficio">Joule Chrome Extension</a>
+      </step>
+      <step>
+        Click the extension icon
+      </step>
+      <step>
+        Click <span class="font-weight-bold">Get started</span>
+      </step>
+      <step>
+        Click <span class="font-weight-bold">Remote node</span>
+      </step>
+      <step>
+         In the <span class="font-weight-bold">"Node URL"</span>, enter the following and click <span class="font-weight-bold">Connect</span>
+         <input-copy class="my-1" :value="https://umbrel.local:8080" auto-width ></input-copy>
+      </step>
+      <step>
+        Chrome will ask to allow access to send requests. Click <span class="font-weight-bold">Allow</span>
+      </step>
+      <step>
+        It will say "Failed to connect to node," and give you a link. Click the link that says <span class="font-weight-bold">"clicking this link"</span>.
```

Comment:
Maybe it's better to just say "open this link"?
```suggestion
        It will say "Failed to connect to node", and give you a link. Open this link.
```